### ISSUE TITLE
NTR: Tetrasaccharide high in urine (oligosaccharides) (Lab: urine tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8840]
New term request Tetrasaccharide high in urine (oligosaccharides) (elthat_101209001357):   UMLS: 2938929

**Comment by pnrobinson** [comment]
@MickeySegal
this seems like a rather imprecise term because what four saccharides are we talking about. I have seen a few older manuscripts but it is a bit unclear. Can you provide context? Is this measured as such today?

**Comment by MickeySegal** [comment]
The term should be "Glucose tetrasaccharide".  The citation has what appears to be an erroneous comma between the two words, which confused our curator, but there is no comma in the 4 cited references.
https://www.ncbi.nlm.nih.gov/books/NBK1261/
"Elevation of the specific urinary glucose, tetrasaccharide, is a highly sensitive finding in IOPD; however, it is also seen in other glycogen storage diseases"

I've asked for a correction in the GeneReviews article.

Synonyms are Glc4 and Hex4.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001008
- label: Elevated urine glucose tetrasaccharide level
- parents: [Oligosacchariduria (HP:0010471)](https://hpo.jax.org/app/browse/term/HP:0010471)
- pmid: PMID:20301438
- def: The amount of glucose tetrasaccharide in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8840
- synonyms:
